ratis-server/src/test/java/org/apache/ratis/statemachine/SimpleStateMachine4Testing.java

```diff
@@ -416,9 +417,15 @@ public void unblockFlushStateMachineData() {
   }
 
   @Override
-  public void notifyFollowerSlowness(RoleInfoProto roleInfoProto) {
-    LOG.info("{}: notifySlowness {}, {}", this, groupId, roleInfoProto);
-    slownessInfo = roleInfoProto;
+  public void notifyFollowerSlowness(RoleInfoProto leaderInfo) {
+    LOG.info("{}: notifySlowness {}, {}", this, groupId, leaderInfo);
+    slownessInfo = leaderInfo;
+  }
```

Comment:
I'm not sure that keeping the deprecated method is okay. Please let me know if you have any opinions.

Reply:
Please remove notifyFollowerSlowness(RoleInfoProto leaderInfo) since it already has notifyFollowerSlowness(RoleInfoProto leaderInfo, RaftPeer slowFollower).  Implementations only have to override one of the methods.